apps/examples/testcase/ta_tc/task_manager/itc/Kconfig

```diff
@@ -0,0 +1,11 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config EXAMPLES_TESTCASE_TASK_MANAGER_ITC
+	bool "Task Manager ITC TestCase Example"
```

Comment:
TC is "TestCase". Let's write full sentence without "ITC"

Reply:
It is similarly written in the UTC kconfig file as
config EXAMPLES_TESTCASE_TASK_MANAGER_UTC
	bool "Task Manager UTC TestCase Example"

Reply:
Ok. That is wrong as well. That can not be a reason to keep duplicated words.